tests/link/test_numba_performance.py

```diff
@@ -0,0 +1,80 @@
+import time
+
+import numpy as np
+import pytest
+
+import aesara.tensor as aet
+from aesara import config
+from aesara.compile.function import function
+from aesara.compile.mode import Mode
+from aesara.compile.sharedvalue import SharedVariable
+from aesara.graph.basic import Constant
+from aesara.graph.fg import FunctionGraph
+from aesara.graph.optdb import OptimizationQuery
+from aesara.link.numba.linker import NumbaLinker
+
+
+opts = OptimizationQuery(include=[None], exclude=["cxx_only", "BlasOpt"])
+numba_mode = Mode(NumbaLinker(), opts)
+py_mode = Mode("py", opts)
+
+
+def set_test_value(x, v):
+    x.tag.test_value = v
+    return x
+
+
+def run_numpy_benchmark(fgraph, numpy_fn, inputs, numpy_inputs, performace_factor=1):
```

Comment:
```suggestion
def run_numpy_benchmark(fgraph, numpy_fn, inputs, numpy_inputs, performance_factor=1):
```
(And correspondingly below.)